README.md

```diff
@@ -1,44 +1,225 @@
+    Learn more about using generic LWC recipes by completing the <a href="https://trailhead.salesforce.com/content/learn/projects/quick-start-lwc-recipes-app">Quick Start: Explore the LWC Recipes Sample App</a> Trailhead project.
+    <br/>
+    <br/>
+    <br/>
+</div>
 
-For example to create an Analytics template from an application run the following to see all the options available:
+> These samples are designed to run on Salesforce Platform.
 
-'sfdx analytics:app:create --help'
+## Table of contents
 
-To view the Analytics template:
+-   [Installing the app using a Scratch Org](#installing-the-app-using-a-scratch-org): This is the recommended installation option. Use this option if you are a developer who wants to experience the app and the code.
 
-'sfdx analytics:template:list'
+-   [Tableau CRM LWC Metadata](#tableau-crm-lwc-metadata)
 
-To create an Analytics application from the template you have two options:
+-   [Tableau CRM LWC Features](#tableau-crm-lwc-features)
 
-From CLI:
+-   [Tableau CRM App Templates](#tableau-crm-app-templates)
 
-'sfdx analytics:app:create -t <templateid>'
+## Installing the app using a Scratch Org
 
-From Analytics Studio
+1. Set up your environment. Follow the steps in the [Quick Start: Lightning Web Components](https://trailhead.salesforce.com/content/learn/projects/quick-start-lightning-web-components/) Trailhead project. The steps include:
 
-> Onetime: set environment variable so you always open to Analytics Page
+    - Enable Dev Hub in your Trailhead Playground
+    - Install Salesforce CLI
+    - Install the Analytics Plugin for the Salesforce CLI
+    - Install Visual Studio Code
+    - Install the Visual Studio Code Salesforce extensions, including the Lightning Web Components ansd Salesforce Analytics - App Templates extensions.
 
-export FORCE_OPEN_URL=/analytics/wave/wave.apexp
+1. If you haven't already done so, authorize your hub org and provide it with an alias (**myhuborg** in the command below):
 
-> Open the scratch org
+    ```
+    sfdx auth:web:login -d -a myhuborg
+    ```
 
-sfdx force:org:open
+1. Clone the forcedotcom/sfdx-analytics repository:
 
-> Select 'Create' > 'App' > 'Start From Template'
+    ```
+    git clone https://github.com/forcedotcom/sfdx-analytics.git
+    cd sfdx-analytics
+    ```
 
-To to view the Analytics applications from the CLI:
+1. Install the Analytics SFDX plugin
+    ```
+    sfdx plugins:install @salesforce/analytics
+    ```
 
-'sfdx analytics:app:list' 
+1. Create a scratch org and provide it with an alias (sfdx-analytics in the command below):
+
+    ```
+    sfdx force:org:create -s -f config/project-scratch-def.json -a sfdx-analytics
+    ```
+
+1. Push the LWC and template sampleas to your scratch org:
+
+    ```
+    sfdx force:source:push
+    ```
+
+1. Update the adminEmail in config/project-scratch-def.json to your email address and open your sc
```

Comment:
The update the adminEmail part should be in in the force:org:create instruction line above.
And, I think `ratch org.` got cut off the end of this sentence.